pkg/pillar/cmd/zedagent/handlenetworkinstance.go

```diff
@@ -7,10 +7,13 @@ package zedagent
 
 import (
 	"bytes"
+	"github.com/golang/protobuf/ptypes/timestamp"
 	"strings"
+	"time"
 
 	"github.com/golang/protobuf/proto"
 	"github.com/golang/protobuf/ptypes"
+	zflow "github.com/lf-edge/eve/api/go/flowlog"
```

Comment:
You can import this as flowlog since that name doesn't conflict and that is more obvious.

Reply:
Sure. Changed to using flowlog